### PR TITLE
Rename LICENSE.txt to LICENSE

### PR DESCRIPTION
**Pull request by CodeMaster7000**
The file should be named `LICENSE` per https://github.com/google/new-project.

### CODE REVIEW

**Comment by mgeisler** [comment]
Hey @CodeMaster7000,

Thanks, you're right that the file should be named `LICENSE` — I see that https://github.com/google/new-project uses that name too.

Can you update https://github.com/google/comprehensive-rust/blob/main/src/credits.md too?

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (mgeisler) [inline LICENSE:1]: Please don't edit the license file — we could keep it exactly like https://github.com/google/new-project/blob/master/LICENSE.

**Review by mgeisler — CHANGES_REQUESTED** [review]
Please update the credits page to link to the new name.

File: LICENSE (hunk 1 above)
Comment (CodeMaster7000) [reply LICENSE:1]: Ok

Comment (CodeMaster7000) [reply LICENSE:1]: This has been done in #272.

Comment (CodeMaster7000) [reply LICENSE:1]: I think both PRs are ready to be merged now.

**Comment by CodeMaster7000** [comment]
I have done the changes that you requested @mgeisler. Since the head repository has been deleted, I can no longer go back and make changes to this PR. However, #272 has the changes so both PRs should be ready to be merged.

**Comment by mgeisler** [comment]
> Since the head repository has been deleted, I can no longer go back and make changes to this PR. However, #272 has the requested changes so both PRs should be ready to be merged.

Thanks for fixing the credits page.

I believe this PR still removes the line at the top of `LICENSE`? Since the repository was deleted, I suggest we close both PRs and you can then open a new single PR which does both changes in one commit (you'll need to clone it down on your own machine for this).

**Comment by CodeMaster7000** [comment]
@mgeisler Refer to #293 for the updated PR that reflects both of those requirements. I have also closed these 2 PRs. Thank you for the feedback!

**Pull request closed without merging**